pulp_rpm/tests/functional/api/test_crud_content_unit.py

```diff
@@ -94,19 +90,71 @@ def test_03_fully_update(self):
 
         This HTTP method is not supported and a HTTP exception is expected.
         """
-        attrs = _gen_content_unit_attrs(self.artifact)
+        attrs = _gen_rpm_package_attrs(self.artifact)
         with self.assertRaises(HTTPError):
             self.client.put(self.content_unit['_href'], attrs)
 
 
-def _gen_content_unit_attrs(artifact):
+def _gen_rpm_package_attrs(artifact):
```

Comment:
This function can be moved to `utils`. It will be re-used by other tests later on.

Reply:
Cool, will do.  I had considered it but I didn't know if the other tests that might use it would be in the same file or not.